tests/testthat/test-extract_features.R

```diff
@@ -26,6 +26,10 @@ patrick::with_parameters_test_that(
       on.exit(parallel::stopCluster(cluster))
     }
 
+    if(skip){
+      skip("skipping whole data test case")
+    }
```

Comment:
Consider moving this to the beggining of the file.

Reply:
I agree!

Reply:
Done in https://github.com/RECETOX/recetox-aplcms/pull/131/commits/1b1ab5ccdba0bb306d5f00bbdd797edd848e7ff0